.circleci/config.yml

```diff
@@ -0,0 +1,88 @@
+---
+version: 2.1
+
+docker_auth: &docker_auth
+  username: $DOCKERHUB_USERNAME
+  password: $DOCKERHUB_PASSWORD
+
+defaults: &defaults
+  docker:
+    - image: greenpeaceinternational/circleci-base:latest
+      auth:
+        <<: *docker_auth
+  working_directory: /home/circleci/
+jobs:
+  clone-tokens-repo:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Clone planet4-design-tokens
+          command: |
+            mkdir tmp && cd "$_"
+            git clone -b PLANET-7002_implement-ci-automation https://github.com/greenpeace/planet4-design-tokens
+      - persist_to_workspace:
+          root: ./tmp/planet4-design-tokens
+          paths:
+            - ""
+  build-tokens:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - attach_workspace:
+          at: ./tmp/planet4-design-tokens
+      - run:
+          name: Convert json tokens into a sass file
+          command: |
+            cd ./tmp/planet4-design-tokens
+            python3 ./bin/build.py
```

Comment:
you can just make the script executable (`chmod +x ./bin/build.py`) and then there is no need to include `python3` here since it's already included in the header of the script. Same for the other two python scripts.